src/lighteval/tasks/lighteval_task.py

```diff
@@ -569,6 +578,15 @@ def load_datasets(tasks: list["LightevalTask"], dataset_loading_processes: int =
                     ],
                 )
 
+        # TODO: debug purpose, to remove later
+        import os
+
+        debug_samples = int(os.getenv("DATASET_SAMPLES", 0))
+        if debug_samples > 0:
+            for dataset in datasets:
+                for split in dataset.keys():
+                    dataset[split] = dataset[split].select(range(debug_samples))
+
```

Comment:
This one should be removed, is there a better way to run in debug mode? I can add it later in a follow-up PR